### PR TITLE
Removed Println Invocations

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Removed Invocations of Println and changed them to logrus calls.

**Related Issues**

https://github.com/kyma-project/kyma-environment-broker/issues/497

### CODE REVIEW

File: common/setup/builder.go [hunk 1]
```diff
@@ -91,7 +91,11 @@ func (b *AppBuilder) WithBrokerClient() {
 }
 
 func (b *AppBuilder) WithProvisionerClient() {
-	b.provisionerClient = provisioner.NewProvisionerClient(b.cfg.Provisioner.URL, b.cfg.Provisioner.QueryDumping)
+	if b.logger == nil {
+		panic("Logger is not defined")
```
Comment (jaroslaw-pieszka) [inline common/setup/builder.go:95]: Is it really the reason to panic?
Needs discussion.

File: internal/model.go [hunk 2]
```diff
@@ -736,9 +737,9 @@ type ConfigForPlan struct {
 	KymaTemplate         string          `json:"kyma-template" yaml:"kyma-template"`
 }
 
-func (c *ConfigForPlan) ContainsAdditionalComponent(componentName string) bool {
+func (c *ConfigForPlan) ContainsAdditionalComponent(componentName string, log logrus.FieldLogger) bool {
 	for _, c := range c.AdditionalComponents {
-		fmt.Println(c.Name)
+		log.Infof("looking for additional component %s", c.Name)
```
Comment (jaroslaw-pieszka) [inline internal/model.go:742]: I think logging here is not necessary at all. 
I would just remove Println.

File: internal/process/provisioning/btp_operator_overrides.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ func (s *BTPOperatorOverridesStep) Run(operation internal.Operation, log logrus.
 	f := func(op *internal.Operation) {
 		op.InstanceDetails.ServiceManagerClusterID = clusterID
 	}
-	if !operation.InputCreator.Configuration().ContainsAdditionalComponent(internal.BTPOperatorComponentName) {
+	if !operation.InputCreator.Configuration().ContainsAdditionalComponent(internal.BTPOperatorComponentName, log) {
```
Comment (jaroslaw-pieszka) [inline internal/process/provisioning/btp_operator_overrides.go:32]: Will this part be removed soon? If so, I would not bother here.

Comment (jaroslaw-pieszka) [inline internal/process/provisioning/btp_operator_overrides.go:32]: But if this part is going to be left in code we should be consistent.
So we should pass `log` or `logger` but not both at the same time.

Comment (ralikio) [reply internal/process/provisioning/btp_operator_overrides.go:32]: Removed.

File: common/setup/builder.go (hunk 1 above)
Comment (ralikio) [reply common/setup/builder.go:95]: I would leave it as it is should never be the case that logger is not defined.

Comment (jaroslaw-pieszka) [reply common/setup/builder.go:95]: I see, but the thing is that you have method WithLogger, and you allow not to call it.
This building method is optional, so if we require logger to be defined, we should make it mandatory.

Comment (ralikio) [reply common/setup/builder.go:97]: I have moved logger instance creation to builder constructor to make it implicit, ok? https://github.com/kyma-project/kyma-environment-broker/pull/654/commits/475c13b5e1d4336fabcdf019c0dd1654e88cc98f

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**